src/pages/export/MetaData.js

```diff
@@ -69,30 +69,21 @@ export class MetaDataExport extends FormBase {
                 sharing,
             } = this.getFormState()
 
-            const params = []
-            params.push('assumeTrue=false')
-            params.push(`format=json`)
-            params.push('download=true')
-            params.push(
-                schemas
-                    .map(name => name)
-                    .sort()
-                    .map(name => `${name}=true`)
-                    .join('&')
-            )
-            params.push(EXCLUDE_PARAMS.map(name => `${name}=false`).join('&'))
-
-            if (sharing !== 'true') {
-                params.push(
-                    'fields=:owner,!user,!publicAccess,!userGroupAccesses'
-                )
-                params.push('skipSharing=true')
-            }
+            let endpoint = `metadata`
+            const downloadUrl = getDownloadUrl({
+                format,
+                compression,
+                endpoint,
+                sharing,
+            })
+            const schemaParams = schemas
+                .map(name => name)
+                .sort()
+                .map(name => `${name}=true`)
+                .join('&')
 
-            let endpoint = `metadata${format}${
-                compression !== 'none' ? compression : ''
-            }`
-            window.location = api.url(`${endpoint}?${params.join('&')}`)
+            const url = downloadUrl.concat(`&${schemaParams}`)
```

Comment:
`downloadUrl` is a string, I would find it clearer if you used a template string here instead of concat.